boilerplate/app/components/Screen.tsx

```diff
@@ -269,6 +269,10 @@ const $outerStyle: ViewStyle = {
   width: "100%",
 }
 
+const $justifyFlexEnd: ViewStyle = {
```

Comment:
It's not in scope for [the bug](https://github.com/infinitered/ignite/issues/2351), but when it's non-scrolling, it's better to add `justify-content: flex-end` to make keyboard avoid behaviour work properly like in this video. 

https://github.com/user-attachments/assets/d9360f5c-e73c-4fff-a342-200f7ff10f54